2nd attempt: Update issue template formatting. (#75)
arfc/saltproc: .github/ISSUE_TEMPLATE/bug_report.md

---

> ## Expected behavior

Reviewer: Using header formatting here is much better!

A clear and concise description of what you expected to happen.

## Describe the bug
A clear and concise description of what the bug is.

## To Reproduce
Steps to reproduce the behavior:
1. 
2. 
3. 
4. 

> ## Screenshots/output

Reviewer: ```suggestion
## Supporting Informatioon
```

> If applicable, add screenshots or program output to help explain your problem.

Reviewer: Maybe we should reorder this to put screenshots last? It's much easier to search by word with formatted quote text rather than screenshots? 

```suggestion
If applicable, add program output with triple backticks, a gist of your output, or a screenshot to help explain your problem.
```

Author: This is an excellent clarification.

## System Specifications:

>  - Software version:

Reviewer: Is this just relevant to saltproc version, or are there required dependencies that we usually have issues with?

Reviewer: e.g. should we ask for serpent versions? openMC versions? numpy?

Author: Yeah we should probably specify.
```suggestion
 <!-- At a minimum, include the SaltProc version and versions of software directly related to the bug-->
 - Software version:
```
